packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx

```diff
@@ -5,7 +5,7 @@ import type { BaseTextProps } from './BaseText';
 import BaseTextComponent from './BaseText';
 
 export default {
-  title: 'Components/Typography/BaseText',
+  title: 'Components/Typography/BaseText (Internal)',
```

Comment:
can you rebase with master and make these changes for other components like Button that we shipped recently?

Reply:
Will do

Reply:
@kamleshchandnani button already has it, BaseText too. We don't have any other internal components I think. 

We can merge this if you want.